src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -175,8 +191,12 @@ class CollapsibleCourseRun extends React.Component {
           break;
       }
     }
-    if (status === PUBLISHED) {
-      return 'Publish Run';
+    if (status === PUBLISHED) return 'Publish Run';
```

Comment:
nit: get outta here with these 1 liners without brackets, I'll swap the linter if not 😁